contracts/test/TestRecipient.sol

```diff
@@ -77,6 +77,7 @@ contract TestRecipient is BaseRelayRecipient, IKnowForwarderAddress {
     //function with no return value (also test revert with no msg.
     function checkNoReturnValues(bool doRevert) public view {
         (this);
+
```

Comment:
no big thing, but I try to leave less files changed by a PR according to git history.

Reply:
ok. will remove.